src/sage/modular/pollack_stevens/padic_lseries.py

```diff
@@ -21,13 +21,15 @@
 # ****************************************************************************
 
 from sage.arith.misc import kronecker, binomial
+from sage.misc.lazy_import import lazy_import
 from sage.rings.integer_ring import ZZ
-from sage.rings.padics.factory import Qp as pAdicField
 from sage.rings.padics.precision_error import PrecisionError
 from sage.rings.power_series_ring import PowerSeriesRing
 from sage.rings.rational_field import QQ
 from sage.structure.sage_object import SageObject
 
+lazy_import('sage.rings.padics.factory', 'Qp', as_='pAdicField')
+
```

Comment:
Why only this is lazy-imported while `PrecisionError` from `sage.rings.padics.precision_error` is just imported?

Reply:
sage.rings.padics.precision_error is shipped by **sagemath-categories**

One cannot do try: ... except ... with a LazyImport